data/linux-bridge/002-linux-bridge.yaml

```diff
@@ -37,8 +37,16 @@ spec:
             - |
               echo 'Installing bridge and tuning CNIs'
               cni_mount_dir=/opt/cni/bin
-              cp --remove-destination /usr/src/github.com/containernetworking/plugins/bin/bridge ${cni_mount_dir}/cnv-bridge || exit 1
-              cp --remove-destination /usr/src/github.com/containernetworking/plugins/bin/tuning ${cni_mount_dir}/cnv-tuning || exit 1
+              sourcebinpath=/usr/src/github.com/containernetworking/plugins/bin
+              cp --remove-destination ${sourcebinpath}/bridge ${cni_mount_dir}/cnv-bridge || exit 1
```

Comment:
consider please to add `-ce` to line 36 and then you don't need `|| exit 1` all over

Reply:
cheking..

Reply:
fixed [here](https://github.com/kubevirt/cluster-network-addons-operator/pull/949/commits/8c4b52ca5b802a31132376ea6390a096180b564e) thanks!